MODULES.md

```diff
@@ -33,23 +33,28 @@ Addon-operator does not use values.yaml as the only source of values for the cha
 
 ## Chart.yaml
 
-We recommend to define the `version` field in your Chart.yaml as "0.0.1" and use VCS to control versions. We also recommend to explicitly specify the `name` field even despite it is ignored: Addon-operator passes the module name to the Helm as a release name.
+We recommend to define the "version" field in your Chart.yaml as "0.0.1" and use VCS to control versions. We also recommend to explicitly specify the "name" field even despite it is ignored: Addon-operator passes the module name to the Helm as a release name.
 
 ## Releases deduplication
 
-A module’s execution might be triggered by an event that does not change the parameters of the module (see [modules discovery](LIFECYCLE.md#modules-discovery)). Re-running Helm will lead to an "empty" release. To avoid this, Addon-operator compares values’ checksums and starts the installation of a Helm chart only if there are some changes.
+A module’s execution might be triggered by an event that does not change the values used by helm templates (see [modules discovery](LIFECYCLE.md#modules-discovery)). Re-running Helm will lead to an "empty" release. To avoid this, Addon-operator runs a command `helm template` and compares a checksum of output with a saved checksum and starts the installation of a Helm chart only if there are some changes.
```

Comment:
```suggestion
A module’s execution might be triggered by an event that does not change the values used by Helm templates (see [modules discovery](LIFECYCLE.md#modules-discovery)). Re-running Helm will lead to an "empty" release. To avoid this, Addon-operator runs a `helm template` command and compares a checksum of output with a saved checksum and starts the installation of a Helm chart only if there are changes.
```